src/server/services/records.js

```diff
@@ -80,8 +80,9 @@ async function recordsForUpload (upload) {
       continue
     }
 
+    const typeRules = rules[type]
```

Comment:
Nit: It took me a second to figure out what this variable was.

Are you comfortable with a more verbose name, like `rulesForCurrentType`?

Reply:
yeah, good call. I'll update this now